app/build.gradle

```diff
@@ -119,7 +119,8 @@ dependencies {
     }
 
     // Nextcloud SSO
-    implementation 'com.github.nextcloud.android-common:ui:0.28.0'
+    implementation 'com.github.nextcloud.android-common:ui:49be0069f6'
```

Comment:
Shall we keep it like that?
Renovate will bump to 0.29.0 once it is released, I assume.